test/parallel/test-require-dot.js

```diff
@@ -15,4 +15,4 @@ m._initPaths();
 
 const c = require('.');
 
-assert.strictEqual(c.value, 42, 'require(".") should honor NODE_PATH');
+assert.strictEqual(c.value, 42, `require(".") should honor NODE_PATH; expected 42, found ${c.value}`);
```

Comment:
this line will be well beyond 80 chars. Can you please format it ? hint: follow @Trott 's suggestion